api/graphql/schema.graphql

```diff
@@ -1043,6 +1045,7 @@ input CreatePoolInput {
     operationalRequirements: [OperationalRequirement] @rename(attribute: "operational_requirements")
     keyTasks: LocalizedStringInput @rename(attribute: "key_tasks")
     status: PoolStatus = NOT_TAKING_APPLICATIONS @rename(attribute: "pool_status")
+    priorityNumber: String
```

Comment:
priorityNumber? probably typo?

Reply:
Haha, yeah. They are both pr...number.

Reply:
In the issue linked to this PR, one of the [screenshots](https://user-images.githubusercontent.com/16977254/184662703-182b9ef8-0737-47d0-a718-c984055ccebb.png) shows a value with hyphens and alphabetical characters and the description of the issue describes it as "a string of characters", so `String` might actually be the correct type here. @Jerryescandon, can you confirm?